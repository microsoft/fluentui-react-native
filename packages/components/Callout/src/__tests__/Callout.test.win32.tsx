import * as React from 'react';
import { Callout } from '..';
import * as renderer from 'react-test-renderer';

it('Callout default props', () => {
  const tree = renderer.create(<Callout />).toJSON();
  expect(tree).toMatchSnapshot();
});
