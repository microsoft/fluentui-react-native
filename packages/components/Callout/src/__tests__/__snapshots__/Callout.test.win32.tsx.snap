// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Callout default props 1`] = `
<RCTCallout
  style={
    Object {
      "backgroundColor": "#faf9f8",
      "borderColor": "#ffffff",
      "borderWidth": 1,
    }
  }
/>
`;
